package.json

```diff
@@ -72,7 +72,7 @@
     "isparta": "4.0.0",
     "istanbul": "0.4.4",
     "json-loader": "0.5.4",
-	"jsdom": "8.5.0",
+	  "jsdom": "8.5.0",
```

Comment:
Looking at the raw it is indented too far. 
https://raw.githubusercontent.com/fayimora/react-slingshot/246df149ce08e70e5267d3d2b60e0be6d72299e3/package.json